src/app/components/table/VirtualTable.jsx

```diff
@@ -442,16 +445,6 @@ export default class VirtualTable extends PureComponent {
     const {
       selectedCell: { selectedCell }
     } = store.getState();
-    const { lastScrolledCell } = this.state;
```

Comment:
I'm not sure if this is still needed or if this is an artifact of the previous version. Couldn't find a reason for it.

Reply:
Hm. Das war mal nötig um einen Infinity-Loop aufzubrechen.